src/main/java/tc/oc/pgm/doublejump/DoubleJumpMatchModule.java

```diff
@@ -97,6 +98,8 @@ public void setKit(Player player, @Nullable DoubleJumpKit kit) {
   public void onPlayerToggleFlight(final PlayerToggleFlightEvent event) {
     Player player = event.getPlayer();
     Jumper jumper = this.jumpers.get(player);
+    if (player.getGameMode() == GameMode.CREATIVE && player.hasPermission(Permissions.DEBUG))
+      return; // HACK: Allow players who are testing to fly
```

Comment:
If a map has creative & double-jump for everyone, it could be working differently because of the perms, wich isn't desirable.

Either you consider people on creative just shouldn't double-jump at all since they can fly, or we don't add this at all.